api/response/schema_query.py

```diff
@@ -68,23 +66,25 @@ def get_room_response(self, room_response, room_id):
                 response.append(response_in_room)
         return (response, total_room_resources)
 
-    @Auth.user_roles('Admin')
-    def resolve_room_response(self, info, room_id):
-        query = Room.get_query(info)
+    def filter_response_by_room(self, info, room_name):
         query_response = Response.get_query(info)
-        room = query.filter_by(id=room_id).first()
+        responses = []
+        room = RoomModel.query.filter(RoomModel.name.ilike(
+                    "%" + room_name + "%")).first()
         if not room:
-            raise GraphQLError("Non-existent room id")
-        room_response = query_response.filter_by(room_id=room_id)
-        responses, total_room_resources = Query.get_room_response(
-            self, room_response, room_id)
+            raise GraphQLError("Room not found")
```

Comment:
We are not searching for the room hence this message is misleading. We are searching for responses to particular existing room questions, therefore, we should get a message telling us that there are no responses to the questions belonging that particular room if it has no responses yet.